### PR TITLE
Feature/daef 482 Simple confirmation dialog for sending money

### PR DESCRIPTION
**Pull request by tomothespian**
This PR introduces simple confirmation dialog for sending money.

## TODOs
- [x] Inject missing Japanese translations
```
"wallet.send.confirmationDialog.addressToLabel": "To",
"wallet.send.confirmationDialog.amountLabel": "Amount",
"wallet.send.confirmationDialog.back": "Back",
"wallet.send.confirmationDialog.feesLabel": "Fees",
"wallet.send.confirmationDialog.submit": "Send",
"wallet.send.confirmationDialog.totalLabel": "Total",
"wallet.send.confirmationDialog.walletPasswordFieldPlaceholder": "Type your spending password",
"wallet.send.confirmationDialog.walletPasswordLabel": "Spending password",
"wallet.send.confirmationDialog.title": "Confirm transaction",
"wallet.send.form.next": "Next",
```

wallet send page
wallet send form confirmation dialog

### CODE REVIEW

**Comment by darko-mijic** [comment]
@a-rukin do we need a title on the confirmation dialog?

**Comment by a-rukin** [comment]
Yes, and close icon too. Title can be "Send Recipe" 
https://zpl.io/boXR0X2

File: app/components/wallet/WalletSendConfirmationDialog.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    .feesWrapper {
+      width: 50%;
+    }
+    .amountLabel,
+    .feesLabel {
+      line-height: 1.38;
+    }
+    .amount,
+    .fees {
+      color: #ea4c5b;
```
Comment (nikolaglumac) [inline app/components/wallet/WalletSendConfirmationDialog.scss]: @tomothespian we can not have hardcoded colors - please use css var here.

File: app/components/wallet/WalletSendConfirmationDialog.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      opacity: 0.5;
+    }
+  }
+
+  .totalAmountLabel {
+    line-height: 1.38;
+  }
+
+  .totalAmount {
+    color: #ea4c5b;
```
Comment (nikolaglumac) [inline app/components/wallet/WalletSendConfirmationDialog.scss]: @tomothespian same thing with the css var here please ;)

File: app/containers/wallet/dialogs/WalletSendConfirmationDialogContainer.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import type { StoresMap } from '../../../stores/index';
+import type { ActionsMap } from '../../../actions/index';
+import WalletSendConfirmationDialog from '../../../components/wallet/WalletSendConfirmationDialog';
+
+@inject('actions', 'stores') @observer
+export default class WalletSendConfirmationDialogContainer extends Component {
+
+  static defaultProps = { actions: null, stores: null };
+
+  props: {
```
Comment (nikolaglumac) [inline app/containers/wallet/dialogs/WalletSendConfirmationDialogContainer.js]: @tomothespian please replace the props with `InjectedDialogContainerProps`.

**Comment by darko-mijic** [comment]
@tomothespian: Please change dialog title to "Confirm transaction".

**Comment by a-rukin** [comment]
Noticed that there is no "from" section of addresses in confirmation dialog.

**Comment by nikolaglumac** [comment]
@a-rukin this is because we actually don't have that information before the transaction is created ;)

**Comment by nikolaglumac** [comment]
@a-rukin title bottom margin reduced to requested 20px:

screen shot 2017-09-16 at 20 01 59

**Comment by DominikGuzei** [comment]
Reviewing this now …

**Comment by DominikGuzei** [comment]
@nikolaglumac im adding the missing translations now …

**Comment by DominikGuzei** [comment]
This PR is ready to be merged when CI is green 👍

**Comment by nikolaglumac** [comment]
@DominikGuzei I have merged latest master and fixed conflicts. We can merge this PR once CI passes.

**Pull request merged**